### DISCUSSION TITLE
2022-32: Github 实用小技巧

### DISCUSSION THREAD

**Discussion by giscus** [Xuanwo/blog#375] [Announcements]
# 2022-32: Github 实用小技巧

An infrastructure engineer, focused on distributed storage system

https://xuanwo.io/reports/2022-32/

**Comment by sunheyi6** [comment]
学到了

**Comment by lizheming** [comment]
typo GitHub →_→

**Reply by Xuanwo** [reply]
修啦，不过标题就不动了，免得 RSS 那边出来一篇新文章

**Comment by Ziy1-Tan** [comment]
太干了！！
